nx.json

```diff
@@ -76,23 +76,7 @@
         "command": "eslint src"
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33019&commitId=d58685a66c4165e33b5c1088647fb494656bda65&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.No beak.chromium.png|2306| Changed|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.Left bottom edge.chromium.png|3022| Changed|
|Callout.Left top edge.chromium.png|1949| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>